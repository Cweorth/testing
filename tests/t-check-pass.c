#include <cut.h>

TEST(checkSuccess) {
    CHECK(1);
}
