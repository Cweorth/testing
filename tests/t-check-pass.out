[  1] checkSuccess...........................................................OK

Summary:
  tests:       1
  succeeded:   1
  skipped:     0
  failed:      0
